src/components/accordion/AccordionItem.jsx

```diff
@@ -28,30 +35,47 @@ const AccordionItem = ({
   titleSize = 'large',
   children,
   className = '',
+  defaultOpened = false,
 }: PropType) => {
   const contentRef = useRef<HTMLDivElement | null>(null);
-  const id = useRef<string>(`AccordionItem_${generateId()}`);
+  const {current: id} = useRef<string>(`AccordionItem_${generateId()}`);
   const [isHover, setIsHover] = useState(false);
 
-  const item = id.current;
-  const {opened, onChange} = useContext(AccordionContext);
-  const isHidden = !opened[item];
+  const {
+    state: {opened, noGapBetweenElements},
+    dispatch,
+  } = useContext(AccordionContext);
+  const isHidden = !opened[id];
 
   const handleClickOnBody = () => {
     if (!isHidden) {
       return;
     }
-    onChange(item, true);
+    handleOpen(true);
     setIsHover(false);
   };
 
   const handleClickOnTitle = () => {
     if (isHidden) {
       return;
     }
-    onChange(item, false);
+    handleOpen(false);
   };
 
+  function handleOpen(value: boolean) {
+    dispatch({
+      type: 'accordion/SET_OPENED',
+      payload: {id, value},
+    });
+  }
+
+  useEffect(() => {
+    if (defaultOpened) {
+      handleOpen(true);
+    }
+    //eslint-disable-next-line
+  }, [defaultOpened]);
```

Comment:
After rethinking we shouldn't use only one prop in this list. Let's keep it empty to indicate our intention (component did mount). Also allowing change of state after component is rendered would be against common behaviour of `default/initial` props